src/veil/ringct/anonwallet.cpp

```diff
@@ -2292,7 +2292,7 @@ int AnonWallet::AddBlindedInputs_Inner(CWalletTx &wtx, CTransactionRecord &rtx,
 
         std::vector<std::pair<MapRecords_t::const_iterator, unsigned int> > setCoins;
         std::vector<COutputR> vAvailableCoins;
-        AvailableBlindedCoins(vAvailableCoins, true, coinControl);
+        AvailableBlindedCoins(vAvailableCoins, true, coinControl, /*nMinimumAmount*/2);
```

Comment:
is the nMinimumAmount needed here? Should it get replaced with the 2?

Reply:
I set it to two because I noticed some 1 sat outputs were being added when I was seeing this error pop up. I have a suspicion that there are some 1 sat outputs floating around that aren't actually 1 sat, and screw up the entire tx.